packages/calcite-components/src/components/text-area/assets/t9n/messages.json

```diff
@@ -1,4 +1,5 @@
 {
   "invalid": "Invalid",
-  "tooLong": "The current character length is {currentLength}, which exceeds the maximum character length of {maxLength}."
+  "tooLong": "The current character length is {currentLength}, which exceeds the maximum character length of {maxLength}.",
+  "tooShort": "The current character length is {currentLength}, which is less than the minimum character length of {minLength}."
```

Comment:
Please refrain from using this message string in component code while translations are pending to avoid errors for non-en locales. 

Currently, we don't provide fallback values for new strings added in existing bundles. Fallback is used only for new components/ new bundles.

Once translations are back one can access the `this.messages.tooShort` prop in component.

cc @jcfranco

Reply:
@anveshmekala Can you create an issue for us to explore an option for us to use messages regardless of translation status?